cmd/pulsard/main.go

```diff
@@ -54,31 +57,28 @@ func parseInputParams() inputParams {
 
 // Need to fix problem with start pulsar
 func main() {
+	_cfg := configuration.NewPulsarConfiguration()
```

Comment:
Why do we need NewPulsarConfiguration() ?
Can we use struct directly, like cfg := PulsarConfiguration{} ?

Reply:
because we are using accessors instead of direct access

Reply:
my point was that NewPulsarConfiguration was added as constructor with defaults, now we don't need it. But for beauty, it's ok )

Reply:
I think using `NewPulsarConfiguration` instead of ` PulsarConfiguration{}` is confusing - as in comment above says it's not clear are we using default value in config or not. May be we should use direct access for clarity?